app/models/identity_header.rb

```diff
@@ -10,8 +10,14 @@ def initialize(b64_identity)
     @b64_identity = b64_identity
   end
 
+  def blank?
+    @b64_identity.blank?
+  end
+
   def valid?
     identity.present? && entitled?
+  rescue JSON::ParserError, NoMethodError
```

Comment:
IMO this should be rescued in the `content` method where parsing occurs so that any call to `content`, `identity`, etc. will be rescued.

Reply:
Yeah, I was thinking about it... `content` could be also covered. Some other issues (like nil content) can bubble up. Lets see if  I can find something better.

Reply:
Moved to `content` with tests added. `NoMethodError` is no longer needed.